es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,58 @@
+
+
+import TodoList from './todo-list';
+import TodoForm from './todo-form';
+import  { createElement } from '../lib/index';
+import { noop } from '../lib/index';
```

Comment:
You can do this with a single import statement, and you don't need to specify the `index` file.
```JAVASCRIPT
import { createElement, noop } from '../lib';
```